.github/workflows/test.yml

```
@@ -279,28 +283,55 @@ jobs:
 
     timeout-minutes: 20
     steps:
-      - name: Download build artifacts
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v4
+
+      - name: Add GNU tar to PATH (significantly faster than windows tar)
+        if: matrix.flavor.target == 'windows'
+        run: echo "C:\Program Files\Git\usr\bin" >> "${{env.GITHUB_PATH}}"
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2028:info:1:6: echo may not expand escape sequences. Use printf

(shellcheck)